tests/error/struct_errors/invalid_generic.err

```diff
@@ -0,0 +1,6 @@
+Guppy compilation failed. Error in file $FILE:14
+
+12:    @guppy.struct(module)
+13:    class MyStruct(Generic[X]):
+                              ^
+GuppyError: Not a parameter
```

Comment:
Could this mention that it's expecting a guppy type var instead of a python one?

Reply:
That would be nice! It's a bigger change since we would need to hold on to the Python scope when defining a struct, but that's probably a good idea anyways

Reply:
Ok, I looked into this and it's not super easy to detect. I'll leave this to another PR (see #215)